challenges/10exercises.js

```diff
@@ -1,26 +1,181 @@
-// Define a function max() that takes two numbers as arguments and returns the largest of them.
-// Use the if-then-else construct available in Javascript.
+// // 1. Define a function max() that takes two numbers as arguments and returns the largest of them.
+function max(number1, number2) {
+  if (number1 > number2) {
+    return number1;
+  }
+  if (number2 > number1) {
+    return number2;
+  }
+  if (number1 === number2) {
+    return "The two numbers are equal."
+  }
+}
+// // Use the if-then-else construct available in Javascript. ??(Wouldn't work for equal)
+//
+// // 2. Define a function maxOfThree() that takes three numbers as arguments and returns the largest of them.
+  function maxOfThree(number1, number2, number3) {
+    return Math.max(number1, number2, number3);
+  }
```

Comment:
You could have used this method above also, if not explicitly asked to do it using the if-then-else structure. I implemented a solution that would actually work for three or more arguments.
```
function maxOfThree( args ) {
 +  let compare = [...arguments][0]
 +
 +  for( let i = 1; i < [...arguments].length; i++ ) {
 +    if ( compare < [...arguments][i] ) {
 +      compare = [...arguments][i]
 +    }
 +  }
 +
 +  return compare